packages/nimble-components/src/accordion/specs/README.md

```diff
@@ -0,0 +1,170 @@
+                            <div slot="default">
+                                <span>Nested Accordion One Content</span>
+                            </div>
+                    <nimble-accordion-item>
+                    <nimble-accordion-item>
+                         <div slot="heading">Nested Accordion Two</div>
+                            <div slot="default">
+                                <span>Nested Accordion Two Content</span>
+                            </div>
+                    <nimble-accordion-item>
+                </nimble-accordion>
+            </div>
+    </nimble-accordion-item>
+    <nimble-accordion-item>
+        <div slot="heading">Accordion four</div>
+            <div slot="default">
+                <nimble-button>Text Button</nimble-button>
+                <span>Accordion one content</span>
+            </div>
+    </nimble-accordion-item>
+</nimble-accordion>
+```
+
+### API
+
+#### Nimble Accordion
+
+[FAST accordion API](https://github.com/microsoft/fast/blob/57f3c22c6341d8a21d48b1ffb7fcbfab1ffd02d8/packages/web-components/fast-foundation/src/accordion/accordion.spec.md)
+
+-   _Component Name:_ `nimble-accordion`
+-   _Properties/Attributes:_
+    -   `appearance` - `block`, `outline`, `ghost`
+-   _Methods:_ Unchanged
+-   _Events:_ Unchanged
+-   _CSS Classes and Custom Properties that affect the component:_ Unchanged
+-   _Slots:_ Unchanged
+
+The Figma design includes appearances of the accordion header that reflect those of the `block`, `outline`, and `ghost` appearances used in other components. These designs are similar to those of the nimble-button and its `appearance` types, but the accordion does not use the `.control` css class, it uses `.region` and `.heading`. Because of this, differences in border functionality, and the possibility of excess overridden css classes if patterns are used, shared styles will not be used, and new styles for the accordion and accordion-item will be created. The appearances of `block`, `outline`, and `ghost` will be implemented under the HTML attribute `appearance`, which will use conditional css styles based on `appearance`'s value (ex. :host([appearance='outline']) will have styling for the outline button).
+
+Documentation will be added to advise against using multiple Nimble Accordions with different appearances next to each other.
+
+#### Nimble Accordion Item
+
+[FAST accordion-item API](https://github.com/microsoft/fast/tree/57f3c22c6341d8a21d48b1ffb7fcbfab1ffd02d8/packages/web-components/fast-foundation/src/accordion-item)
+
+-   _Component Name:_ `nimble-accordion-item`
+-   _Properties/Attributes:_
+    -   `error-visible` - boolean
+-   _Methods:_ Unchanged
+-   _Events:_ Unchanged
+-   _CSS Classes and Custom Properties that affect the component:_ Unchanged
+-   _Slots:_ The `start` slot will not be used based on the visual designs, but the `end` slot will be used to display the error icon in the `error-visible` accordion-item state.
```

Comment:
It would probably be good to include `collapsed-icon` and `expanded-icon` here. I'm guessing the `nimble-accordion-item` will set the content of those slots and it would be unsupported to override those icons?

Reply:
Sure, I'll include them. If I understand your question correctly, it is supported to override the `collapsed-icon` and `expended-icon`- I've been looking into the accordion item implementation in Nimble, and I was able to set these icons to nimble icons in the export of the accordion item component in index.ts [See FAST's example regarding this](https://github.com/microsoft/fast/tree/archives/fast-element-1/packages/web-components/fast-foundation/src/accordion#accordionitem).